macros/edr/tests/test_utils/clean_elementary_test_tables.sql

```diff
@@ -1,17 +1,19 @@
 {% macro clean_elementary_test_tables() %}
     {% set test_table_relations = [] %}
-    {% set temp_test_table_relations_map = elementary.get_cache("temp_test_table_relations_map") %}
-    {% for test_entry in temp_test_table_relations_map.values() %}
-        {% for test_relation in test_entry.values() %}
-            {% do test_table_relations.append(test_relation) %}
+    {% set temp_test_table_relations_map = elementary.get_cache("temp_test_table_relations_map", {}) %}
```

Comment:
If you are checking `if temp_test_table_relations_map` there is no need to return a default value of `{}` 🙂 
```suggestion
    {% set temp_test_table_relations_map = elementary.get_cache("temp_test_table_relations_map") %}
```